pkgs/by-name/ne/nekoray/nixos-disable-setuid-request.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/src/global/NekoGui.cpp b/src/global/NekoGui.cpp
+index 7943d7a..2095a15 100644
+--- a/src/global/NekoGui.cpp
++++ b/src/global/NekoGui.cpp
+@@ -355,10 +355,16 @@ namespace NekoGui {
+     // System Utils
+ 
+     QString FindNekoBoxCoreRealPath() {
+-        auto fn = QApplication::applicationDirPath() + "/nekobox_core";
+-        auto fi = QFileInfo(fn);
+-        if (fi.isSymLink()) return fi.symLinkTarget();
+-        return fn;
++        // find in PATH first
++        QString path = QStandardPaths::findExecutable("nekobox_core");
++        if (!path.isEmpty()) {
++            return path;
++        } else {
++            auto fn = QApplication::applicationDirPath() + "/nekobox_core";
++            auto fi = QFileInfo(fn);
++            if (fi.isSymLink()) return fi.symLinkTarget();
++            return fn;
```

Comment:
A bit of a nitpick, but you could shorten this patch by not putting this part into an `else`, since we return at the end of the `if` anyways.